osc-ui/src/main/java/org/osc/core/broker/view/maintenance/NetworkLayout.java

```diff
@@ -170,36 +147,6 @@ public void buttonClick(ClickEvent event) {
         return this.editNATSettings;
     }
 
-    @SuppressWarnings("serial")
-    private void editIPSettingsClicked() throws Exception {
-        try {
-            if (!hasDeployedInstances()) {
-                ViewUtil.addWindow(new SetNetworkSettingsWindow(this, this.setNetworkSettingsService));
-            } else {
-                final VmidcWindow<OkCancelButtonModel> alertWindow = WindowUtil.createAlertWindow(
-                        VmidcMessages.getString(VmidcMessages_.NW_CHANGE_WARNING_TITLE),
```

Comment:
remove all messages which are not applicable

Reply:
I need little more clarifications.

Reply:
you have messages in the *.properties files which are no longer used, we need to remove them see /osc-ui/src/main/resources/org/osc/core/broker/view/common/VmidcMessages.properties

Reply:
There is no extra message as part of static ip configurations.Still,the existing messages pointing the same DHCP and NAT messages.So,nothing to be removed.Please let me know if you see any specific messages to be removed.

Reply:
got it. the message is still used in NAT settings...so we are good.